Makefile

```
@@ -0,0 +1,11 @@
+test:
+	cargo build
+	cargo test
+
+fmt:
+	deno run -A --unstable tools/format.js
```

Could also be

```suggestion
  cargo run -- run -A --unstable tools/format.js
```

To fmt/lint with the result of cargo build.

---

That's not the behaviour in CI, these are meta scripts which are meant to use the latest release.